admin/class-convertkit-pmp-admin.php

```diff
@@ -465,16 +465,36 @@ public function after_all_membership_level_changes( $pmpro_old_user_levels ) {
 			// Build a unique array of tags to subscribe to contact and remove from contact.
 			$subscribe_tags = array_diff( $new_tags, $old_tags );
 			$unsubscribe_tags = array_diff( $old_tags, $new_tags );
-		
+
 			// Get the subscriber information.
 			$user = get_userdata( $user_id );
 			$user_email = $user->user_email;
 			$user_name = $user->first_name . ' ' . $user->last_name;
 
+			/**
+			 * Allow custom code to filter the subscribe tags for the user by email.
+			 *
+			 * @param array $subscribe_tags The array of tag IDs to subscribe this email address to.
```

Comment:
```suggestion
			 * @since TBD
			 *
			 * @param array $subscribe_tags The array of tag IDs to subscribe this email address to.
```

Reply:
We can use whatever version we need here instead of TBD, feel free to update that.